compliance/aws/ecs_unused/AWS_Unused_ECS_Clusters.pt

```diff
@@ -0,0 +1,399 @@
+name "AWS Unused ECS Clusters"
+rs_pt_ver 20180301
+type "policy"
+short_description "Report and remediate any ECS clusters that are not currently in use. \n See the [README](https://github.com/rightscale/policy_templates/tree/master/Compliance/aws/ecs_unused) and [docs.rightscale.com/policies](http://docs.rightscale.com/policies/) to learn more."
+long_description "Version: 1.0"
+category "Compliance"
+severity "low"
+
+###############################################################################
+# Permissions
+###############################################################################
+
+permission "perm_read_creds" do
+  actions "rs_cm.show_sensitive","rs_cm.index_sensitive"
+  resources "rs_cm.credentials"
+end
+
+###############################################################################
+# User inputs
+###############################################################################
+
+parameter "param_email" do
```

Comment:
change the label and add the description 
```
parameter "param_email" do

  type "list"

  label "Email addresses to notify"

  description "Email addresses of the recipients you wish to notify when new incidents are created"

end
```

Reply:
@cdwilhelm 
Updated with latest parameters pattern